File: C++/BubbleSort.cpp
#include<iostream>
#include<vector>
using namespace std;

int main(){
    int ar[10] = {3,15,78,2,45,19,8,12,40,55};
    int size = 10,temp = 0;
    for(int i=0;i<size;i++){
        for(int j=0;j<size-i-1;j++){
            if(ar[j]>ar[j+1]){
                temp = ar[j+1];
                ar[j+1] = ar[j];
                ar[j] = temp;
            }
        }
    }

    for(int item : ar)
        cout<<" "<<item;
    return 0;
}

File: Python/BubbleSort.py
def swap(x,y):
    x = x + y
    y = x - y
    x = x - y

def BubbleSort(ar):
    size = len(ar)
    for x in range(size):
        for y in range(size-x-1):
            if ar[y] > ar[y+1]:
                temp = ar[y]
                ar[y] = ar[y+1]
                ar[y+1] = temp
    return ar

def main():
    ar = [12,77,98,45,8,5,44,36,15,50]
    BubbleSort(ar)
    print(ar)

if __name__ == '__main__':
    main()